[debops.icinga_db] Set fact interpreter to python3

imrejonk opened the pull request:
The icinga_db fact script fails when run with Python 2. Because of this, the icinga_web role does not set up the database backend correctly, resulting in a broken installation. This commit changes the interpreter to Python 3 which works fine.

This is the error I got when running the fact script with Python 2:
```
Traceback (most recent call last):
  File "/etc/ansible/facts.d/icinga_db.fact", line 50, in <module>
    output.update(get_config(config_file))
  File "/etc/ansible/facts.d/icinga_db.fact", line 25, in get_config
    config.read_string(config_string)
AttributeError: ConfigParser instance has no attribute 'read_string'
```

imrejonk committed:
[debops.icinga_db] Set fact interpreter to python3

The icinga_db fact script fails when run with Python 2. This commit changes the interpreter to Python 3.

drybjed commented:
I wonder if that should be a preferred approach for solving cases like this in the future. I know that Python3 will be the default in the future, but there are still old installations out there that might not have it.

@imrejonk: how about adding another set of `try/except` clause in this case that, when `read_string` is not available, reverts to using `readfp` from the Python2 version of ConfigParser? On Python3 it will hopefully will not be reached at all, and on Python2 it should allow the script to work fine in this case. Want to try implementing it? You can check https://github.com/debops/debops/commit/c5a8bd1a0960a92d7acf2f4ec1aa6c134c23508c to see what was used previously here.

imrejonk committed:
Revert "[debops.icinga_db] Set fact interpreter to python3"

This reverts commit 82b6fa60a601abc4c907190ba33f8b776a90a160.

imrejonk committed:
[debops.icinga_db] Make fact script Py2 compatible

The debops.icinga_db fact script was no longer Python 2 compatible since
commit c5a8bd1a0960a92d7acf2f4ec1aa6c134c23508c. These changes ensure that the
script can be used with both Python 2 and Python 3.

imrejonk commented:
Well okay then, let's maintain these scripts for both Python 2 and 3 a little longer...
How do you like this solution? We are already using it in production :)

imrejonk committed:
[debops.icinga_db] Catch only AttributeName exception

drybjed commented:
@imrejonk Much better, thanks. :-) Just fix the issue on Travis-CI and it will be ready for merge.

imrejonk committed:
[debops.icinga_db] Properly catch ImportError

imrejonk commented:
Crap, forgot the first Except statement. Travis is unforgiving!

drybjed reviewed (approved):
Nice work, thanks. :+1:
